engine/lib/views.php

```diff
@@ -1622,6 +1624,12 @@ function elgg_view_list_item($item, array $vars = array()) {
 		return elgg_view_river_item($item, $vars);
 	}
 
+	$view = elgg_extract('item_view', $vars);
```

Comment:
If `item_view` is set to an existing view, shouldn't we use it instead of calling the type-specific functions? So this would come first in the function.